test/common/testlib.py

```diff
@@ -2037,7 +2037,10 @@ def enable_root_login(self):
 
         By default root login is disabled in cockpit, removing the root entry of /etc/cockpit/disallowed-users allows root to login.
         """
-        self.sed_file('/root/d', '/etc/cockpit/disallowed-users')
+
+        # fedora-coreos runs cockpit-ws in a containter so does not install cockpit-ws on the host
+        if not self.machine.ostree_image:
+            self.sed_file('/root/d', '/etc/cockpit/disallowed-users')
```

Comment:
same, please move into first commit. Unrelated to "erase rpms before installing on rhel8"